Makefile

```diff
@@ -16,6 +16,7 @@ K8SRELEASEDIR=$(shell echo "$(K8SRELEASE)" | sed "s/\./_/g")
 
 APISRC=gen-apidocs
 APIDST=$(WEBROOT)/static/docs/reference/generated/kubernetes-api/v$(K8SRELEASE)
+APILATEST=$(shell ( echo v$(K8SRELEASE); ls -d $(dir $(APIDST))/v[0-9]* ) | xargs basename -a | sort -V -r | head -1)
```

Comment:
Hi @joelsmith .
Just catching up and trying to put the pieces together. A couple of thoughts:
The gist of this change is to update the proposed `latest` entry in the website/static/_redirects file.
This target would be run after the docs are released?
I guess I have reservations about updating the _redirects file from a target in this Makefile.
I wonder if the docs can provide a more permanent solution by creating a `latest` page for you to 
reference? That way there is no need to modify the _redirects file.
I have not checked the targets.
APIDST should be set to something like:
`website/static/docs/reference/generated/kubernetes-api/v1.18`
I have not sorted out the details, but what do you think about creating a `latest` directory (or page)?